.github/ISSUE_TEMPLATE.md

```diff
@@ -11,6 +11,7 @@
 <!--- Provide a link to a live example on JSFiddle or Codepen -->
 
 ### Your environment
+* Angular/angular-cli version:
```

Comment:
Maybe just `Angular version:`?

`Handsontable version:` below, could be helpful for investigation.

Reply:
Ok